src/util.ts

```diff
@@ -236,6 +236,7 @@ export class ApiError extends Error {
     }
 
     this.message = ApiError.createMultiErrorMessage(errorBody, this.errors);
+    Error.captureStackTrace(this);
```

Comment:
@bcoe thoughts on this one....?

Reply:
to me this seemed like a reasonable fix, I tend to use `Error.stack` more than `Error.message` when debugging.